LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Neo Financial Technologies Inc., Dirk-Jan Rutten
```

Comment:
```suggestion
Copyright (c) 2020 Neo Financial Technologies Inc.
Copyright (c) 2017 Dirk-Jan Rutten
```

Reply:
I don't actually know the correct way to do this but some Googling resulted in this.